toil_orchestrator/tasks.py

```diff
@@ -12,6 +12,7 @@
 
 
 logger = logging.getLogger(__name__)
+RETRY_NUMBER = 5
```

Comment:
I thought this option is low level enough to be set in the file, but I am open for this to be moved to settings.py